cmd/helm/package.go

```diff
@@ -139,6 +141,11 @@ func (p *packageCmd) run() error {
 		debug("Setting version to %s", p.version)
 	}
 
+	if len(p.appVersion) != 0 {
```

Comment:
it's probably simpler to just check `if p.appVersion != ""` instead of casting to an int.

Reply:
Thank you for the review. I have updated the PR.